docs/getting-started/index.md

```diff
@@ -14,15 +20,19 @@ If you are not sure where you should get started, check out our **Guided User St
 
 ## Products
 
-Within the Substrate ecosystem, you can build blockchains, smart contracts, and even user interfaces! Here is a high level overview of the different products which comprise these space:
+Within the Substrate ecosystem, you can build blockchains, smart contracts, and even user interfaces! Here is a quick overview of the different products which comprise these spaces:
```

Comment:
I think it should be singular "space"

```suggestion
Within the Substrate ecosystem, you can build blockchains, smart contracts, and even user interfaces! Here is a quick overview of the different products which comprise this space:
```

Reply:
Here `these spaces` are used to refer to `blockchains, smart contracts, and even user interfaces`, should it be plural? I'm not an English native speaker, correct me if I'm wrong.

Reply:
The word "space" already has some reference to a "bunch of things". Usually things grouped in a category. I would use "space" to refer to all things blockchain. I would use "spaces" to refer to different kinds of topics like blockchains, cars, computers, etc...